src/routes/exercise-finished/ExerciseFinishedScreen.tsx

```diff
@@ -185,6 +183,7 @@ const ExerciseFinishedScreen = ({ navigation, route }: Props): ReactElement => {
           <ShareSection disciplineTitle={disciplineTitle} results={results} />
         </Root>
       </RouteWrapper>
+      {Platform.OS === 'ios' && <RouteWrapper shouldApplyFullscreen={false} />}
```

Comment:
So we have to wrappers here for ios? One of which wraps nothing? Why not just use one wrapper like this:
```suggestion
      <RouteWrapper shouldApplyFullscreen={Platform.OS === 'android'} />
```
At least some comment should be added why this is necessary.